docs/shop/companysettings/orderlimits.rst

```diff
@@ -0,0 +1,30 @@
+Order Ceiling
+=============
+
+The order ceiling defines an order limit for users per month. It can only be set
+or changed by a shop admin in the user management section of the shop. If an
+user has an order ceiling set and tries to order more than the set sum the shopping
```

Comment:
... and the sum of all orders exceeds this limit, the shopping cart will be disabled. Moreover, the webshop-admin of the user is going to be notified by email and then has do accept or decline the order, before the user is going to be able to continue shopping.